roles/lib_openshift/src/test/integration/oc_configmap.yml

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/ansible-playbook --module-path=../../../library/
+## ./oc_configmap.yml -M ../../../library -e "cli_master_test=$OPENSHIFT_MASTER
+---
+- hosts: "{{ cli_master_test }}"
+  gather_facts: no
+  user: root
+  vars:
+    filename: /tmp/test_configmap_from_file
+
+  post_tasks:
+  - name: Setup a file with known contents
+    copy:
+      content: This is a file
+      dest: "{{ filename }}"
+
+  - name: create a test project
+    oc_project:
+      name: test
+      description: for tests only
+
+  ###### create test ###########
+  - name: create a configmap
+    oc_configmap:
+      state: present
+      name: configmaptest
+      namespace: test
+      from_file:
+        config: "{{ filename }}"
+      from_literal:
+        foo: bar
+
+  - name: fetch the created configmap
+    oc_configmap:
+      name: configmaptest
+      state: list
+      namespace: test
+    register: cmout
+
+  - debug: var=cmout
+
+  - name: assert configmaptest exists
+    assert:
+      that:
+      - cmout.results.results[0].metadata.name == 'configmaptest'
+      - cmout.results.results[0].data.foo == 'bar'
+  ###### end create test ###########
+
+  ###### update test ###########
```

Comment:
Can we also add an update test to verify that we can overwrite existing keys correctly? (Just for sanity)